tasks/gen/resource/browser.cr

```diff
@@ -148,13 +149,11 @@ class Lucky::ResourceTemplate < Teeplate::FileTree
   directory "#{__DIR__}/../templates/resource"
 
   getter resource, columns
-  getter operation_filename : String,
-    query_filename : String,
+  getter query_filename : String,
     underscored_resource : String,
     folder_name : String
 
   def initialize(@resource : String, @columns : Array(Lucky::GeneratedColumn))
-    @operation_filename = operation_class.underscore
```

Comment:
It looks like the `operation_class` method on line 174 can be removed.

Reply:
I see that it is used in other resource ecr's. I will remove it and replace the occurrences with Save<%= resource %>